cmd/cmd.go

```diff
@@ -122,6 +126,15 @@ func (r *runner) rootRunE(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return errors.Wrap(err, "cmd: get flag --auto-assign")
 	}
+	isWorkspace, err := cmd.PersistentFlags().GetBool(flagWorkspace)
```

Comment:
```suggestion

	isWorkspace, err := cmd.PersistentFlags().GetBool(flagWorkspace)
```
Please add a newline here.